commit d14c61821dc7935dcc9feaedeb051fd5e5d8463f
Author: bertin <macportsuser@macports.org>
Date:   Thu May 22 23:41:25 2025 +0200

    /opt/local/site-ports/sysutils/MacPorts/files/port-o-option-default.diff

diff --git a/src/port/port.tcl b/src/port/port.tcl
old mode 100644
new mode 100755
index 3c7151949..41f39c54a
--- a/src/port/port.tcl
+++ b/src/port/port.tcl
@@ -4333,6 +4333,8 @@ proc parse_options { action ui_options_name global_options_name } {
 
     set options_order(${action}) {}
 
+    # RJVB
+    set global_options(ports_ignore_different) yes
     while {[moreargs]} {
         set arg [lookahead]
 
